unit-tests/utilities/number/stabilized/test-change-percent.cpp

```diff
@@ -0,0 +1,42 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+//#cmake:add-file ../../../../common/utilities/number/stabilized-value.h
+
+
+#include "../../../test.h"
+#include <../common/utilities/number/stabilized-value.h>
+
+using namespace utilities::number;
+
+// Test group description:
+//       * This tests group verifies stabilized_value class.
+//
+// Current test description:
+//       * Verify that on the same values different 'get' calls with different stabilization percent
+//         give the expected value
+TEST_CASE( "change stabilization percent", "[stabilized value]" )
+{
+    try
+    {
+        stabilized_value< float > stab_value( 5 );
+        CHECK_NOTHROW( stab_value.add( 1.0f ) );
+        CHECK_NOTHROW( stab_value.add( 1.0f ) );
+        CHECK_NOTHROW( stab_value.add( 1.0f ) );
+        CHECK_NOTHROW( stab_value.add( 1.0f ) );
+        CHECK_NOTHROW( stab_value.add( 1.0f ) );
+
+        CHECK_NOTHROW( stab_value.add( 2.0f ) );
+        CHECK_NOTHROW( stab_value.add( 2.0f ) );
+        CHECK_NOTHROW( stab_value.add( 2.0f ) );
+        CHECK_NOTHROW( stab_value.add( 2.0f ) );
+
+        CHECK( 1.0f == stab_value.get( 1.0f ) );
```

Comment:
Why?? 2 is at 80%... There is no stable value at 100%...

Reply:
Same answer as before. We give the best result we can we do not return a failure.
You think it's better to return a failure on this case?

Reply:
Same answer as before